src/main/java/com/alipay/oceanbase/rpc/table/AbstractTableQueryImpl.java

```diff
@@ -31,6 +31,8 @@
 public abstract class AbstractTableQueryImpl extends AbstractTableQuery {
 
     protected ObTableQuery tableQuery;
+    // TableEntry key
+    protected String indexTableName;
```

Comment:
Noticed that you added indexTableName to both AsyncImpl and StreamImpl, have you ever tested both of them? Or do we only need StreamImpl?

Reply:
I have added both of them for the reason that indexTableName parameter pass in class AsyncImpl is easier and less modify as a class member by this way. I have tested both Query and QueryAsync interface